src/api/attachments.tsx

```diff
@@ -24,3 +30,43 @@ export const usePostAttachmentMetadata = (): UseMutationResult<
       postAttachmentMetadata(attachmentMetadata),
   });
 };
+
+export const getAttachment = async (id: string): Promise<AttachmentPostMetadataResponse> => {
+  return storageApi.get(`/attachments/${id}`).then((response) => {
+    return response.data;
+  });
+};
+
+export const useGetAttachment = (
+  id: string
+): UseQueryResult<AttachmentPostMetadataResponse, AxiosError> => {
+  return useQuery({
+    queryKey: ['Attachment', id],
+    queryFn: () => getAttachment(id),
+  });
+};
+
+const getAttachments = async (
+  entityId: string
+): Promise<AttachmentPostMetadataResponse[]> => {
+  const queryParams = new URLSearchParams();
+  queryParams.append('entity_id', entityId);
+  queryParams.append('modified_time', '2024-01-02T13:10:10.000+00:00')
+  queryParams.append('created_time', '2024-01-01T12:00:00.000+00:00')
```

Comment:
These shouldn't be added as the backend doesn't have these filters (at least yet).

Reply:
yeah, now I made that change in object storage API, I can write `created_time` and `modified_time` straight into `mocks/attachment.json` I think.